### PR TITLE
Changing the caption for the GitHubTeacher slide

### PR DESCRIPTION
**Pull request by agoila**

### CODE REVIEW

**Comment by LuLuKotewicz** [comment]
🐶

File: _posts/0000-01-20-agoila.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
 layout: slide
-title: "Welcome to our slide deck!"
+title: "Ankit Goila's slide"
```
Comment (LuLuKotewicz) [inline _posts/0000-01-20-agoila.md:3]: I love that you changed the title. Wish I thought of that!

File: _posts/0000-01-20-agoila.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
 ---
 
 ![sad-puppy](https://cloud.githubusercontent.com/assets/16547949/25401096/b9e619ee-29c2-11e7-8ee3-74a301737d34.jpg)
 {: .center}
 
-Caption here
+I'm sorry. It was me.
```
Comment (LuLuKotewicz) [inline _posts/0000-01-20-agoila.md:9]: Cute!

Comment (LuLuKotewicz) [reply _posts/0000-01-20-agoila.md:9]: @agoila I'm mentioning you here so you get a ping!

**Review by LuLuKotewicz — COMMENTED** [review]
@agoila I'm mentioning you again because I put the first in the wrong place!

**Review by beardofedu — APPROVED** [review]

**Pull request merged**